Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+COPY humioctl /humioctl
```

Comment:
With Docker [multistage builds](https://docs.docker.com/develop/develop-images/multistage-build/) you don't have to build `humioctl` prior to building the docker image. And you can have Docker Hub do it all for you :)

Reply:
In this case we're shipping binaries to other places such as homebrew, so we have to build the binaries anyway. I've never used goreleaser but apparently it's able to figure out the docker image depends on the binary so it works out the order of operations. I agree though typically the build of the binary should be done in docker... Maybe we will move that way if it becomes hard to do this with drone  (haven't gotten that far yet).